lib/Chat/SpecialRoom/Manager.php

```diff
@@ -72,13 +72,13 @@ public function userHasNewChangelog(string $userId): bool {
 	public function createNotesIfNeeded(string $userId): void {
 		$room = $this->roomManager->getSpecialRoom($userId, Room::NOTES_CONVERSATION);
 
-		if ($this->getNotesForUser($userId) === (int) 0) {
+		if ($this->getNotesForUser($userId) === 0) {
```

Comment:
I think this will fail. It will think 0 is a string afaik. I originally had == which worked just fine but then lint complained.

Reply:
Lint complains because `==` is illegal nowerdays :speak_no_evil: 

```php
public function getNotesForUser(string $userId): int {
```

it will always return an integer (you defined it like that correctly) and so you need to compare it to a integer `0` is always an integer, `'0'` would be a string.

Reply:
When I had it that way the comparison result was always false :-/ Maybe it was just my mistake.